watson-developer-cloud/discovery-components
# feat: structured query convert inputs

#### What do these changes do/fix?
These changes add in the conversion and display of the query that matches the selections made in the `StructuredQuery` component.

Note that I didn't add Cypress tests in this PR, as I think it makes sense to add those in in a separate issue.

#### How do you test/verify these changes?
You can take a look at and run the tests. You can also play around with the `StructuredQuery` component in Storybook, selecting fields, operators, and entering values and watching the copyable query at the bottom of the page update. Confirm that the query looks as you would expect it to look.

#### Have you documented your changes (if necessary)?
not in this PR, there is another issue to update the documentation for the `StructuredQuery` component

#### Are there any breaking changes included in this pull request?
no
